core/src/main/scala/quasar/datasource/kafka/KafkaConsumerBuilder.scala

```diff
@@ -84,6 +86,24 @@ object KafkaConsumerBuilder {
     (record: CommittableConsumerRecord[F, Array[Byte], Array[Byte]]) =>
       Stream.chunk(Chunk.bytes(Option(record.record.value).getOrElse(Array.empty)))
 
+  def AsJson[F[_]]: RecordDecoder[F, Array[Byte], Array[Byte]] =
```

Comment:
Why do we need to add a new decoder type `AsJson` that includes both the key and value?

Reply:
I can remove it for sure, but at least for me it seems pretty weird that we can't have both keys and values inside data analysis. What's the point of having only keys or only values? If I send something like `key: 2021-01-26:22:34  value: { message: "trololo"}` and then `key: 2021-01-26:22:36 value: {message: "trololo"}` the information about this message was sent twice is erased, it's just `Ok, two`. I can remove it if it's not needed though.

Reply:
That makes sense. Let's keep this here for now. I'll try to find out exactly what the JSON on the Kafka topic will look like for this initial customer use case.

Reply:
When we initially implemented the Kafka datasource, we didn't have a concrete use case in mind, and so we decided to support both JSON and CSV, at the cost of only decoding the key or the value, exclusive. Now that we have a use case, we can let it shape the decision we make here.